azkaban-common/src/main/java/azkaban/executor/ExecutionRampDao.java

```diff
@@ -70,23 +71,19 @@ public ExecutableRampMap handle(final ResultSet resultSet) throws SQLException {
             ExecutableRamp.createInstance(
                 resultSet.getString(1),
```

Comment:
Can you please convert the hardcoded numbers into meaningful enum?

Reply:
The number is mapping with SQL statement well. And, all other DAO in azkaban is using the same pattern. But, sure, if you have concern, let's address the problem in a seperate PR.